espresso-tests/migration-test.bash

```diff
@@ -50,51 +51,64 @@ CHILD_CHAIN_UPGRADE_EXECUTOR_ADDRESS=$(cast call $L1_TOKEN_BRIDGE_CREATOR_ADDRES
 PRIVATE_KEY="$(docker compose run scripts print-private-key --account l2owner | tail -n 1 | tr -d '\r\n')"
 OWNER_ADDRESS="$(docker compose run scripts print-address --account l2owner | tail -n 1 | tr -d '\r\n')"
 
-# Echo for debug
-echo "Deploying Espresso Osp"
-# Change directory to orbit actions dir for the following commands.
 cd $ORBIT_ACTIONS_DIR
-# ** Essential migration step ** Forge script to deploy new OSP entry. We do this to later point the rollups challenge manager to the espresso integrated OSP.
-forge script --chain $PARENT_CHAIN_CHAIN_ID contracts/parent-chain/espresso-migration/DeployEspressoOsp.s.sol:DeployEspressoOsp --rpc-url $PARENT_CHAIN_RPC_URL --broadcast -vvvv
+forge update
+echo "Deploying mock espresso tee verifier"
+forge script --chain $PARENT_CHAIN_CHAIN_ID ../espresso-tests/DeployMockVerifier.s.sol:DeployMockVerifier --rpc-url $PARENT_CHAIN_RPC_URL --broadcast -vvvv
+
+ESPRESSO_TEE_VERIFIER_ADDRESS=$(cat broadcast/DeployMockVerifier.s.sol/1337/run-latest.json | jq -r '.transactions[0].contractAddress' | cast to-checksum)
```

Comment:
the use of chain ID 1337 suggests this setup is primarily for local testing. Do we ever test with a fork and thus need the chain id to change?

Reply:
This is just a local testing thing. As are the other chain ID's. This test wasn't even written to be tested on forks of other chains, but I suppose that could be done in the future if it gives us better testing.

Reply:
okie, could be a future issue then but sounds low priority